build.gradle

```diff
@@ -1,31 +1,40 @@
 buildscript {
   repositories {
+    mavenLocal()
+    google()
     jcenter()
   }
+  dependencies {
+    classpath "com.android.tools.build:gradle:$project.androidBuildToolsVersion"
+    classpath "org.aspectj:aspectjtools:$project.aspectjVersion"
+    classpath "com.github.dcendents:android-maven-gradle-plugin:$project.androidMavenVersion"
+  }
 }
 
-allprojects {
-  buildscript {
-    repositories {
-      jcenter()
-      mavenCentral()
-    }
-  }
+subprojects {
   repositories {
+    maven {
+      url "http://dl.bintray.com/android10/maven"
+    }
     mavenLocal()
-    mavenCentral()
     jcenter()
+    google()
   }
 }
 
-def buildSrcDir = 'buildsystem'
-apply from: "${buildSrcDir}/inject.gradle"
+if (JavaVersion.current().isJava8Compatible()) {
+  allprojects {
+    tasks.withType(Javadoc) {
+      options.addStringOption('Xdoclint:none', '-quiet')
+    }
+  }
+}
 
 task wrapper(type: Wrapper) {
   gradleVersion = project.gradleVersion
 }
 
 task runUnitTests(
-    dependsOn: [':frodo2-logger:test']) {
+        dependsOn: [':frodo2-logger:test']) {
```

Comment:
I would style this, this way:

```
task runUnitTests(dependsOn: [':frodo2-logger:test']) {
 ...
}
```

Just to keep consistency with the rest of gradle tasks